pkg/plugin/conntrack/_cprog/conntrack.c

```diff
@@ -148,10 +181,14 @@ static __always_inline bool _ct_handle_udp_connection(struct packet *p, struct c
     new_value.flags_seen_tx_dir = p->flags;
     new_value.last_report_tx_dir = now;
     new_value.traffic_direction = _ct_get_traffic_direction(observation_point);
+    new_value.conntrack_metadata.packets_forward_count = 1;
```

Comment:
And also i thought we will be putting this behind a configmap or IFDEF ? do you think the impact will be less ?

Reply:
This has been included in bdbcd8f8b3a9bff9a0836b37be0d7b6409a5ad7e